tests/kernel_bundle/use_kernel_bundle_second_queue_ctx_not_equal_to_kernel_bundle_ctx.cpp

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  The test checks that calling sycl::handler::use_kernel_bundle with a kernel
+//  bundle which is associated with a context different from one associated with
+//  a handler results in exception. This is done for kernel bundles that:
+//    1) contain user kernels
+//    2) contain builtin kernels
+//    3) don't contain any kernel
+//
+//  The test verifies that the exception with sycl::errc::invalid was thrown.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "use_kernel_bundle.h"
+#include "use_kernel_bundle_queue_ctx_not_equal_to_kernel_bundle_ctx.h"
+
+#define TEST_NAME \
+  use_kernel_bundle_second_queue_ctx_not_equal_to_kernel_bundle_ctx
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+using namespace sycl_cts::tests::use_kernel_bundle;
+using namespace use_kernel_bundle_queue_ctx_not_equal_kb_ctx;
+
+class TEST_NAME : public sycl_cts::util::test_base {
+ public:
+  /** return information about this test
+   */
+  void get_info(test_base::info &out) const override {
+    set_test_info(out, TOSTRING(TEST_NAME), TEST_FILE);
+  }
+
+  /** execute the test
+   */
+  void run(util::logger &log) override {
+    auto platforms = sycl::platform::get_platforms();
+    const std::vector<sycl::device> devices{sycl::device::get_devices()};
+    if (devices.size() <= 1) {
+      log.note("Test skipped due to one device was found");
```

Comment:
There was a comment on this on another PR

Reply:
I've applied this comment [here](https://github.com/KhronosGroup/SYCL-CTS/pull/225/commits/e157b8bd72174f5b18b4e387b9dfb4e353573bd7#diff-d4520edbb0ca7bc678e694703cae85b14192ee2dd0d8ce877de37be0d7668537L42-L43)